extensions-core/namespace-lookup/src/main/java/io/druid/server/namespace/NamespacedExtractionModule.java

```diff
@@ -79,6 +80,7 @@ public void setupModule(SetupContext context)
           {
             context.registerSubtypes(NamespacedExtractor.class);
             context.registerSubtypes(ExtractionNamespace.class);
+            context.registerSubtypes(NamespaceLookupExtractorFactory.class);
```

Comment:
this is getting to be a long list as well, can it just use `new SimpleModule("DruidNamespacedExtractionModule").registerSubtypes(...)`?